lib/core_dom/resource_url_resolver.dart

```diff
@@ -160,13 +160,11 @@ class ResourceUrlResolver {
     // If it's not absolute, then resolve it first
     Uri resolved = baseUri.resolve(uri);
 
-    // If it's package-relative, tack on 'packages/' - Note that eventually
-    // we may want to change this to be '/packages/' to make it truly absolute
+    // If it's package-relative, tack on '/packages/'
     if (resolved.scheme == 'package') {
-      return 'packages/${resolved.path}';
+      return '/packages/${resolved.path}';
```

Comment:
This makes sense (i.e. for the package scheme.)  In test/_specs.dart, why does the TEST_SERVER_BASE_PREFIX need to be made absolute?